app/views/jobs/_form.html.erb

```diff
@@ -24,14 +24,13 @@
     <div class="col">
       <div class="form-group">
         <label for="description">Art der Anstellung *</label>
-        <%= text_field_tag(
-              "job_payload_employment_type",
-              (@job.payload || {})["employmentType"],
-              required: true,
-              placeholder: "Vollzeit, Teilzeit, Ausbildung, ...",
-              class: "form-control",
-              name: "job[payload][employment_type]"
-            )%>
+        <% @categories = @categories.select { |category| category.tag_list.split(",").include?("generic_item_job") } %>%
```

Comment:
this must not be a @-variable on the left side, right? and `.map(&:name)` can be done here already instead of in the `options_for_select`, right?
and what if `@categories` on the right side does not exist? will the `select` fail with an error?

Reply:
- fixed global variable definition
- fixed map of name

load_category_list is part of before_action in JobsController. if it would be missing or an error, there would be a crash, yes. 
Since the category is a mandatory field, interception without a category is not necessary in my opinion